client/src/pages/Upload.tsx

```diff
@@ -1,16 +1,124 @@
-import { Link } from "react-router-dom";
+import LogoBar from "../components/LogoBar";
+import React, { useState } from "react";
+import Button from "../components/Button";
+import axios from "axios";
+import clsx from "clsx";
+import { useNavigate } from "react-router-dom";
 
 export default function Upload() {
+  const navigate = useNavigate(); 
+
+  const [outlineUploadLoading, setOutlineUploadLoading] = useState(false);
+  const supportedFileTypes = [
+    "application/pdf",
+  ];
+
+  const handleOutlineUpload = async (event?: React.ChangeEvent<HTMLInputElement>) => {
+    if (event!.target.files!.length > 1) {
```

Comment:
This `if` condition is inappropriate, please fix it.

<img width="896" alt="Screenshot 2022-12-08 at 1 25 09 PM" src="https://user-images.githubusercontent.com/20573623/206560392-56f6d4a7-11dc-443f-9d12-5eb17203485c.png">